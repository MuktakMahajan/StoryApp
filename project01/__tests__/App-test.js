/**
 * @format
 */

 import 'react-native';
 import React from 'react';
 import Screen from '../App/components/screen1';
 import Adapter from 'enzyme-adapter-react-16';
 import renderer from 'react-test-renderer';
import Enzyme from 'enzyme';

/**
 * Set up Enzyme to mount to DOM, simulate events,
 * and inspect the DOM in tests.
 */
Enzyme.configure({ adapter: new Adapter() });
 // Note: test renderer must be required after react-native.
 
 
 test('Screen snapShot',()=>{
   const snap = renderer.create(
     <Screen/>
   ).toJSON();
   expect(snap).toMatchSnapshot();
 });

 test('object assignment', () => {
  const data = {one: 1};
  data['two'] = 2;
  expect(data).toEqual({one: 1, two: 2});
});