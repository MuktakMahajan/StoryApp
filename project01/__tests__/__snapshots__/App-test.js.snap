// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Screen snapShot 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#ffffff",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <View
    accessible={true}
    focusable={false}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "display": "flex",
        "flexDirection": "row",
        "justifyContent": "center",
        "paddingHorizontal": 8,
        "paddingVertical": 8,
      }
    }
  >
    <View>
      <Image
        source={
          Object {
            "testUri": "../../../App/components/screen1/profile.jpg",
          }
        }
        style={
          Object {
            "height": 200,
            "marginHorizontal": 3,
            "width": 200,
          }
        }
      />
      <View
        style={
          Array [
            Object {
              "backgroundColor": "transparent",
              "borderRightColor": "transparent",
              "borderRightWidth": 60,
              "borderStyle": "solid",
              "borderTopColor": "#ffffff",
              "borderTopWidth": 60,
              "height": 0,
              "position": "absolute",
              "width": 0,
            },
            undefined,
          ]
        }
      />
      <View
        style={
          Array [
            Object {
              "alignSelf": "flex-end",
              "backgroundColor": "transparent",
              "borderRightColor": "transparent",
              "borderRightWidth": 60,
              "borderStyle": "solid",
              "borderTopColor": "#ffffff",
              "borderTopWidth": 60,
              "height": 0,
              "position": "absolute",
              "transform": Array [
                Object {
                  "rotate": "90deg",
                },
              ],
              "width": 0,
            },
            undefined,
          ]
        }
      />
      <View
        style={
          Array [
            Object {
              "backgroundColor": "transparent",
              "borderRightColor": "transparent",
              "borderRightWidth": 60,
              "borderStyle": "solid",
              "borderTopColor": "#ffffff",
              "borderTopWidth": 60,
              "bottom": 0,
              "height": 0,
              "position": "absolute",
              "right": 0,
              "transform": Array [
                Object {
                  "rotate": "180deg",
                },
              ],
              "width": 0,
            },
            undefined,
          ]
        }
      />
      <View
        style={
          Array [
            Object {
              "backgroundColor": "transparent",
              "borderRightColor": "transparent",
              "borderRightWidth": 60,
              "borderStyle": "solid",
              "borderTopColor": "#ffffff",
              "borderTopWidth": 60,
              "bottom": 0,
              "height": 0,
              "left": 0,
              "position": "absolute",
              "transform": Array [
                Object {
                  "rotate": "270deg",
                },
              ],
              "width": 0,
            },
            undefined,
          ]
        }
      />
    </View>
  </View>
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "opacity": 1,
      }
    }
  >
    <View>
      <Image
        source={
          Object {
            "testUri": "../../../App/components/screen1/images.png",
          }
        }
        style={
          Object {
            "borderRadius": 12,
            "height": 25,
            "width": 25,
          }
        }
      />
    </View>
  </View>
  <Text
    style={
      Object {
        "color": "black",
        "fontSize": 20,
      }
    }
  >
     User-1
  </Text>
  <Text
    style={
      Object {
        "color": "black",
        "fontSize": 20,
      }
    }
  >
     Details-1 
  </Text>
</View>
`;
